dev/io.openliberty.org.jboss.resteasy.cdi/bnd.bnd

```diff
@@ -42,35 +42,37 @@ instrument.disabled: false
   META-INF=resources/META-INF
 
 -buildpath: \
-  org.jboss.resteasy:resteasy-cdi;strategy=exact;version=${resteasy-version},\
-  org.jboss.resteasy:resteasy-servlet-initializer;strategy=exact;version=${resteasy-version},\
-  com.ibm.ws.org.apache.commons.io,\
-  com.ibm.ws.org.apache.httpcomponents,\
-  com.ibm.websphere.javaee.annotation.1.3,\
-  com.ibm.websphere.javaee.cdi.2.0,\
-  com.ibm.websphere.javaee.interceptor.1.2,\
-  com.ibm.websphere.javaee.jaxb.2.2,\
-  com.ibm.websphere.javaee.jaxrs.2.1,\
-  com.ibm.websphere.javaee.jsonp.1.1,\
-  com.ibm.websphere.javaee.servlet.4.0,\
-  com.ibm.websphere.javaee.validation.2.0;version=latest,\
-  com.ibm.websphere.org.eclipse.microprofile.config.1.4;version=latest,\
-  com.ibm.websphere.org.osgi.core;version=latest,\
-  com.ibm.websphere.org.osgi.service.component;version=latest,\
-  com.ibm.ws.adaptable.module;version=latest,\
-  com.ibm.ws.anno;version=latest,\
-  com.ibm.ws.cdi.interfaces;version=latest,\
-  com.ibm.ws.classloading;version=latest,\
-  com.ibm.ws.container.service;version=latest,\
-  com.ibm.ws.container.service.compat;version=latest,\
-  com.ibm.ws.jaxrs.2.0.common;version=latest,\
-  com.ibm.ws.logging.core,\
-  com.ibm.ws.org.jboss.logging,\
-  com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
-  com.ibm.ws.webcontainer;version=latest,\
-  com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
-  io.openliberty.org.jboss.resteasy.common,\
-  javax.activation:activation;version=1.1
\ No newline at end of file
+	org.jboss.resteasy:resteasy-cdi;strategy=exact;version='${resteasy-version}',\
+	org.jboss.resteasy:resteasy-servlet-initializer;strategy=exact;version='${resteasy-version}',\
+	com.ibm.ws.org.apache.commons.io,\
+	com.ibm.ws.org.apache.httpcomponents,\
+	com.ibm.websphere.javaee.annotation.1.3,\
+	com.ibm.websphere.javaee.cdi.2.0,\
+	com.ibm.websphere.javaee.interceptor.1.2,\
+	com.ibm.websphere.javaee.jaxb.2.2,\
+	com.ibm.websphere.javaee.jaxrs.2.1,\
+	com.ibm.websphere.javaee.jsonp.1.1,\
+	com.ibm.websphere.javaee.servlet.4.0,\
+	com.ibm.websphere.javaee.validation.2.0;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.config.1.4;version=latest,\
+	com.ibm.websphere.org.osgi.core;version=latest,\
+	com.ibm.websphere.org.osgi.service.component;version=latest,\
+	com.ibm.ws.adaptable.module;version=latest,\
+	com.ibm.ws.anno;version=latest,\
+	com.ibm.ws.cdi.interfaces;version=latest,\
+	com.ibm.ws.classloading;version=latest,\
+	com.ibm.ws.container.service;version=latest,\
+	com.ibm.ws.container.service.compat;version=latest,\
+	com.ibm.ws.jaxrs.2.0.common;version=latest,\
+	com.ibm.ws.logging.core,\
+	com.ibm.ws.org.jboss.logging,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	com.ibm.ws.webcontainer;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	io.openliberty.org.jboss.resteasy.common,\
+	javax.activation:activation;version='1.1',\
+	com.ibm.ws.org.jboss.weld3
```

Comment:
```suggestion
	javax.activation:activation;version='1.1'
```
Why is weld dependency added to the buildpath?  I don't see any reference to weld classes in the ResteasyCdiExtension.  The other resteasy.cdi projects for ee 10 and 11 do not have weld added to their buildpath.

Reply:
I had pulled in additional classes and missed removal of this when I removed those other classes.